modules/rte_mis_lottery/rte_mis_lottery.module

```diff
@@ -70,3 +76,17 @@ function rte_mis_lottery_views_pre_render(ViewExecutable $view) {
     }
   }
 }
+
+/**
+ * Implements hook_batch_alter().
+ */
+function rte_mis_lottery_batch_alter(&$batch) {
+  // Check if the operations is set.
+  $operations = $batch['sets'][0]['operations'];
+
+  // If the current batch is `lottery result`.
+  if (!empty($operations) && !empty($operations[0][1]) && $operations[0][1][0] == 'lottery_results') {
```

Comment:
@RohanSingh-inno The safe check if still missing.

Reply:
Fixed.